src/app/features/public-form/public-form.module.ts

```diff
@@ -0,0 +1,108 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { RouterService } from "../../core/config/dynamic-routing/router.service";
+import { ViewConfig } from "../../core/config/dynamic-routing/view-config.interface";
+import { EntityDetailsConfig } from "../../core/entity-details/EntityDetailsConfig";
+import { EntityListConfig } from "../../core/entity-list/EntityListConfig";
+import { AdminOverviewService } from "../../core/admin/admin-overview/admin-overview.service";
+import { PublicFormConfig } from "./public-form-config";
+import { AsyncComponent, ComponentRegistry } from "app/dynamic-components";
+
+/**
+ * Configure publicly accessible forms for users without login to record some data into the system.
+ */
+@NgModule({
+  declarations: [],
+  imports: [CommonModule],
+})
+export class PublicFormModule {
+  static databaseEntities = [PublicFormConfig];
+
+  constructor(
+    components: ComponentRegistry,
+    routerService: RouterService,
+    adminOverviewService: AdminOverviewService,
+  ) {
+    components.addAll(dynamicComponents);
+    routerService.addRoutes(viewConfigs);
+    adminOverviewService.menuItems.push({
+      label: $localize`:admin menu item:Manage Public Forms`,
+      link: PublicFormConfig.route,
+    });
+  }
+}
+
+const dynamicComponents: [string, AsyncComponent][] = [
+  [
+    "EditPublicFormField",
+    () =>
+      import(
+        "app/features/public-form/edit-public-form-field/edit-public-form-field.component"
+      ).then((c) => c.EditPublicFormFieldComponent),
```

Comment:
I would rename this to EditPublicFormFields or maybe EditPublicFormColumns. We use the term "form field" for each individual form control already (also within the entity details etc.)

Reply:
can you also rename the file and folder names accordingly?